lms/security.py

```diff
@@ -66,47 +67,60 @@ def get_route_policy(self, request):
         return self.fallback_policy()
 
 
-class LTISecurityPolicy:
+class _LTIUserBasedSecurityPolicy:
     @classmethod
-    def authenticated_userid(cls, request):
-        if request.lti_user is None:
-            return None
+    def get_lti_user(cls, request) -> LTIUser:
+        raise NotImplemented()
+
+    def permits(self, request, context, permission):
+        return _permits(self, request, context, permission)
+
+    def remember(self, request, userid, **kw):
+        pass
+
+    def forget(self, request):
+        pass
 
+    @staticmethod
+    def get_userid(lti_user):
```

Comment:
This method deals exclusively with these bytes/string transformation from and `lti_user`

Reply:
Is `get_userid()` only used internally by this base class? If so it should maybe be `_get_userid()`.